lms/djangoapps/grades/tests/test_new.py

```diff
@@ -90,10 +89,14 @@ def test_course_grade_feature_gating(self, feature_flag, course_setting):
         # Grades are only saved if the feature flag and the advanced setting are
         # both set to True.
         grade_factory = CourseGradeFactory(self.request.user)
+        set_persistent_grades_feature_flags(
+            global_flag=feature_flag,
+            enabled_for_all_courses=False,
+            course_id=self.course.id,
+            enabled_for_course=course_setting
+        )
```

Comment:
NIT/FUTURE: makes sense for this to be a context manager so tests can do `with persistent_grades_feature_flags`.